src/components/modal/Modal.scss

```diff
@@ -133,3 +135,33 @@
     z-index: 89;
     display: flex;
 }
+
+//Responsive support
+.is-responsive-web {
+    .modal {
+        @include breakpoint($modal-fullscreen) {
+            padding: 0;
+        }
+    }
+
+    .modal-dialog-container {
+        width: 100%;
+        margin: auto;
+
+        @include breakpoint($modal-fullscreen) {
+            height: 100%;
+
+            .modal-dialog {
+                min-height: 100%;
+                padding: 22px 16px 12px 16px;
```

Comment:
Why did we choose these specific values? It seems like they'd create inconsistent spacing. Can we keep the 30px padding from the default experience, instead?

Reply:
These were pulled from the Zeplin designs.